echopype/calibrate/calibrate_azfp.py

```diff
@@ -82,23 +88,35 @@ def compute_range_meter(self, cal_type):
             'Sp' for calculating point backscattering strength
         """
         # Notation below follows p.86 of user manual
-        N = self.echodata.vendor['number_of_samples_per_average_bin']  # samples per bin
-        f = self.echodata.vendor['digitization_rate']  # digitization rate
-        L = self.echodata.vendor['lockout_index']  # number of lockout samples
-        sound_speed = self.env_params['sound_speed']
-        bins_to_avg = 1   # keep this in ref of AZFP matlab code, set to 1 since we want to calculate from raw data
+        N = self.echodata.vendor["number_of_samples_per_average_bin"]  # samples per bin
+        f = self.echodata.vendor["digitization_rate"]  # digitization rate
+        L = self.echodata.vendor["lockout_index"]  # number of lockout samples
+        sound_speed = self.env_params["sound_speed"]
+
+        # keep this in ref of AZFP matlab code,
+        # set to 1 since we want to calculate from raw data
+        bins_to_avg = 1
 
         # Calculate range using parameters for each freq
-        #  This is "the range to the centre of the sampling volume for bin m" from p.86 of user manual
-        if cal_type == 'Sv':
+        # This is "the range to the centre of the sampling volume
+        # for bin m" from p.86 of user manual
+        if cal_type == "Sv":
             range_offset = 0
         else:
-            range_offset = sound_speed * self.echodata.beam['transmit_duration_nominal'] / 4  # from matlab code
-        range_meter = (sound_speed * L / (2 * f)
-                       + sound_speed / 4 * (((2 * (self.echodata.beam.range_bin + 1) - 1) * N * bins_to_avg - 1) / f
-                                            + self.echodata.beam['transmit_duration_nominal'])
-                       - range_offset)
-        range_meter.name = 'range'  # add name to facilitate xr.merge
+            range_offset = (
+                sound_speed * self.echodata.beam["transmit_duration_nominal"] / 4
+            )  # from matlab code
+        range_meter = (
+            sound_speed * L / (2 * f)
+            + sound_speed
+            / 4
+            * (
+                ((2 * (self.echodata.beam.range_bin + 1) - 1) * N * bins_to_avg - 1) / f
+                + self.echodata.beam["transmit_duration_nominal"]
+            )
```

Comment:
this section would be more readable as:
```
+ sound_speed / 4 * (
    ((2 * (self.echodata.beam.range_bin + 1) - 1) * N * bins_to_avg - 1) / f
    + self.echodata.beam["transmit_duration_nominal"]
)
```
Is there anything explicit that will make this fail the pre-commit?

Reply:
If you want this to be like that, then we'd have to make `black` ignore it. I think there is a line-length issue.

Reply:
How about adding parentheses: `(sound_speed / 4)`? I have a hunch that would keep those two elements in one line.

(It'd be great if there was an online tool where you could run `black` on code fragments, the way you can test regex's and such)

Reply:
I think `(sound_speed / 4)` is a good idea